beater/beatertest/result.go

```diff
@@ -0,0 +1,23 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package beatertest
+
+import "fmt"
+
+// ResultErrWrap wraps given input into the expected result error string
+func ResultErrWrap(s string) string { return fmt.Sprintf("{\"error\":\"%+v\"}\n", s) }
```

Comment:
there are a lot of shallow files in `beatertest`, can't these functions be packed together?

Reply:
I try to avoid having a file `utility` or similar where it is hard to keep overview and easy to just stuff things in. If you don't feel strongly about this, I'd rather have shallow files here.